docs/tr/week03/03-1.md

```diff
@@ -181,9 +184,7 @@ Example: Suppose the input $x$ is one dimensional and has size of 100 and $w$ ha
 ## Derin Evrişimli Sinir Ağları (DCNNs)
 <!--
 As previously described, deep neural networks are typically organized as repeated alternation between linear operators and point-wise nonlinearity layers. In convolutional neural networks, the linear operator will be the convolution operator described above. There is also an optional third type of layer called the pooling layer.
-
```

Comment:
This empty lines show where a new paragraph begins. Why have you removed them all?